src/composer/scenes/Composer/Composer.tsx

```diff
@@ -598,6 +607,39 @@ class Composer extends React.Component<IProperties, IState> {
     });
   }
 
+  /**
+   * Returns component by id
+   * @param id
+   * @return {}
+   */
+  public getComponentById(id: number, cont?: IContent) {
+    const content = cont || this.state.content;
+    if (!Array.isArray(content.content)) {
+      return {};
+    }
+
+    let c = {} as IContent | LooseObject; // TODO: fix!!
```

Comment:
You can move this variable into a body of for cycle because you never read/write it out of for cycle.

Reply:
I would define that variable every iteration? Really?